src/components/task-step/all-steps.cjsx

```diff
@@ -109,29 +109,29 @@ ExternalUrl = React.createClass
   displayName: 'ExternalUrl'
   mixins: [StepMixin, StepFooterMixin]
   hideContinueButton: -> true
+  getDefaultProps: ->
+    redirectToUrl: (url) -> window.open( url, '_blank')
```

Comment:
Because it needs to be called as an onClick hander which first marks the task as complete, and then redirect.  The redirect fn is a prop so the spec can override the implementation to ensure it's called, but it does have to be triggered from the `onClick`.  

Although! Looking at the implementation below, I think onContinue should be called before the redirect.   I'll swap those line's order.